react/features/dynamic-branding/reducer.js

```diff
@@ -58,6 +58,14 @@ const DEFAULT_STATE = {
      */
     defaultBranding: true,
 
+    /**
+     * Url for a custom page for DID numbers list.
+     *
+     * @public
+     * @type {string}
+     */
+    didPageUrl: '',
```

Comment:
Why not have it as undefined?

Reply:
I went with the same pattern with the rest of the URL strings there. Shall I change everywhere?

Reply:
Consistency is better, let's keep it!